src/modules/landing_target_estimator/python_derivation/SymForceKF_coupled_static.py

```diff
@@ -0,0 +1,108 @@
+import symforce.symbolic as sf
+
+def generate_px4_function(function_name, output_names):
+    from symforce.codegen import Codegen, CppConfig
+    import os
+    import fileinput
+
+    codegen = Codegen.function(
+            function_name,
+            output_names=output_names,
+            config=CppConfig())
+    metadata = codegen.generate_function(
+            output_dir="src/modules/landing_target_estimator/python_derivation/generated/coupled_static",
+            skip_directory_nesting=True)
+
+    print("Files generated in {}:\n".format(metadata.output_dir))
+    for f in metadata.generated_files:
+        print("  |- {}".format(os.path.relpath(f, metadata.output_dir)))
+
+    # Replace cstdlib and Eigen functions by PX4 equivalents
+    with fileinput.FileInput(os.path.abspath(metadata.generated_files[0]), inplace=True, backup='.bak') as file:
```

Comment:
```suggestion
    with fileinput.FileInput(os.path.abspath(metadata.generated_files[0]), inplace=True) as file:
```

those backup files aren't useful, please also don't add them in the PR

Reply:
I've removed the .bak files https://github.com/PX4/PX4-Autopilot/pull/20653/commits/ee5b28adf3c9e53fb88a30aa256eaafe64404fb7